stubs/CMakeLists.txt

```diff
@@ -14,6 +14,16 @@ ENDIF()
 
 PROJECT(@lib_name@)
 
+# check for and enable c++11 support
+INCLUDE(CheckCXXCompilerFlag)
+CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
+IF(COMPILER_SUPPORTS_CXX11)
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
what is the status there ?
@scopatz do you still want me to this changeset into `conda-forge/cyclus-feedstock` ? If so, please tell me more, I don't understand what I should change, and PR.


btw, I am not sure this is relevant as `-stdlib=libc++` is set by default on MAC when using apple clang ):
> On OS X and FreeBSD libc++ is the default standard library and the -stdlib=libc++ is not required.
from: http://libcxx.llvm.org/docs/UsingLibcxx.html.

so this should fulfill the condo-forge requirement even without the `-stdlib=libc++` flag